spec/src/modules/quizzes.js

```diff
@@ -329,15 +361,6 @@ describe(`ConstructorIO - Quizzes${bundledDescriptionSuffix}`, () => {
       return expect(quizzes.getQuizResults('invalidQuizId', { answers: validAnswers })).to.eventually.be.rejected;
     });
 
-    it('Should be rejected if an invalid versionId is provided', () => {
```

Comment:
Why did we get rid of this test?

Reply:
version_id use to reject requests if the versionId was some random stuff, but when we pass it in quiz_version_id it doesn't reject it anymore, there is no error.

Reply:
I see. Thanks

Reply:
Let's keep this test around and update to use `quiz_version_id`. I think it should be returning an error, but it's not. Seems like it might be a bug as mentioned in a previous test above. ([Relevant Slack Thread](https://constructor.slack.com/archives/C04M09VKNNB/p1680292190255649?thread_ts=1679436486.711039&cid=C04M09VKNNB))